charts/graph-node/templates/services.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: graph-node
+  namespace: {{ .Values.global.namespaceOverride | default .Release.Namespace }}
+spec:
+  selector:
+    app: graph-node
+  ports:
+    - name: http
+      port: {{ .Values.graphNode.ports.http | default 8000 }}
```

Comment:
We shouldn't need to specify defaults here if we provide defaults in the value file